src/Common/tests/System/Collections/IDictionary.Generic.Tests.cs

```diff
@@ -155,22 +155,28 @@ protected override void AddToCollection(ICollection<KeyValuePair<TKey, TValue>>
         /// <summary>
         /// Returns a set of ModifyEnumerable delegates that modify the enumerable passed to them.
         /// </summary>
-        protected override IEnumerable<ModifyEnumerable> ModifyEnumerables
+        protected override IEnumerable<ModifyEnumerable> GetModifyEnumerables(ModifyOperation operations)
         {
-            get
+            if ((operations & ModifyOperation.Add) == ModifyOperation.Add)
             {
                 yield return (IEnumerable<KeyValuePair<TKey, TValue>> enumerable) =>
                 {
                     IDictionary<TKey, TValue> casted = ((IDictionary<TKey, TValue>)enumerable);
                     casted.Add(CreateTKey(12), CreateTValue(5123));
                     return true;
                 };
+            }
+            if ((operations & ModifyOperation.Insert) == ModifyOperation.Insert)
+            {
                 yield return (IEnumerable<KeyValuePair<TKey, TValue>> enumerable) =>
                 {
                     IDictionary<TKey, TValue> casted = ((IDictionary<TKey, TValue>)enumerable);
                     casted[CreateTKey(541)] = CreateTValue(12);
                     return true;
                 };
+            }
+            if ((operations & ModifyOperation.Remove) == ModifyOperation.Remove)
+            {
                 //// [ActiveIssue(31112)]
```

Comment:
It should be possible to uncomment this now, and close 31112 (change top description of PR to include "Fixes 31112")

Reply:
Done